test/support/commands.js

```diff
@@ -98,7 +98,7 @@ Cypress.Commands.add('visitOnClock', (url, options = {}) => {
 
   // NOTE: this is a hack to fix the clock around defer around Backbone.history.loadUrl
   cy
-    .wait(10)
+    .wait(200)
```

Comment:
When running locally I think because vite is bundling on the fly, this wait has to be a bit longer.  Unfortunate, but overall I'm not seeing a huge difference.. we'll see on CI I suppose if this has a negative affect.. I'm not sure how we get around this.